hedera-node/src/main/java/com/hedera/services/files/store/BytesStoreAdapter.java

```diff
@@ -64,6 +65,23 @@ public int size() {
 		return delegate.size();
 	}
 
+	@Override
```

Comment:
Why did we add `equals()` and `hashCode()` here? This class won't ever be used in a collection anywhere, right?

Reply:
There was a sonar code smell stating , `Extend a class that overrides equals and add fields without overriding equals in the subclass, you run the risk of non-equivalent instances of your subclass being seen as equal, because only the superclass fields will be considered in the equality test`. I believe that is because this class is extended from `AbstractMap`.  What do you think ?

Reply:
removed them. Ty!